### DISCUSSION TITLE
Run without prompts

### DISCUSSION THREAD

**Discussion by redricka** [grahampugh/erase-install#333] [Q&A] [answered]
Hey, 
Looking at running this through jamf self service, however no matter what tags I have tried to put in, it still prompts me for either a username or password to continue.. Is there a way to make this completely free of those prompts once clicking the instlal button on the Jamf Self Sevice? Thank you

Apologies if its a dumb question.

**Comment by grahampugh** [comment, marked as answer]
https://github.com/grahampugh/erase-install/wiki/6.-Upgrading-or-reinstalling-macOS#apple-silicon-mac
